beginner_source/transfer_learning_tutorial.py

```diff
@@ -111,6 +111,8 @@ def imshow(inp, title=None):
     mean = np.array([0.485, 0.456, 0.406])
     std = np.array([0.229, 0.224, 0.225])
     inp = std * inp + mean
+    inp = np.maximum(inp, 0)
```

Comment:
How about modifying to `inp = np.clip(inp, 0, 1)`?

Reply:
done :)